lib/Doctrine/DBAL/Driver/Statement.php

```diff
@@ -19,6 +19,8 @@
 
 namespace Doctrine\DBAL\Driver;
 
+use Doctrine\DBAL\DBALException;
```

Comment:
You have to revert all of the changes in this file. We cannot change the interface as of `2.x`. The point of this PR is to make the code comply with the current contract.